src/main_menu.cpp

```diff
@@ -359,8 +359,19 @@ bool main_menu::opening_screen()
     print_menu( w_open, 0, iMenuOffsetX, iMenuOffsetY );
 
     if( !assure_dir_exist( FILENAMES["config_dir"] ) ) {
-        popup( _( "Unable to make config directory. Check permissions." ) );
-        return false;
+        if( query_yn( _( "Unable to make config directory.\nTry to create parents?" ) ) ) {
```

Comment:
This should be the automatic behavior of the `assure_dir_exists`. So it applies to all calls to the function.

Reply:
So `assure_dir_exists` should always create the parent directories? If so should I keep the popup asking the user if they would like them created?

Reply:
Just create them, the function is named `assure_dir_exists`, so it should "assure that the directory exists". If you only add this for this one directory, we'll get another report for the next directory that has no existing parent.

Reply:
Thanks for the help! I have added the parent creation to `assure_dir_exists` I tested a couple cases on linux and it seems to work